src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/PackageAnnotationCheck.java

```diff
@@ -48,6 +48,19 @@
  * <pre>
  * &lt;module name=&quot;PackageAnnotation&quot;/&gt;
  * </pre>
+ * <p>
+ * Examples of validating files package-info.java and MyClass.java:
+ * </p>
+ * <p>package-info.java</p>
+ * <pre>
+ * &#64;Deprecated
+ * package com.example.annotations.packageannotation; //ok
+ * </pre>
+ * <p>MyClass.java</p>
+ * <pre>
+ * &#64;Deprecated
+ * package com.example.annotations.packageannotation; //violation
```

Comment:
Please sure another example of `MyClass` to show how to make violation go away (IE: remove annotation).

Reply:
Done